lib/fluent/plugin/out_stdout.rb

```diff
@@ -25,6 +25,9 @@ class StdoutOutput < Output
     DEFAULT_LINE_FORMAT_TYPE = 'stdout'
     DEFAULT_FORMAT_TYPE = 'json'
 
+    desc "If Fluentd logger outputs logs to a file, this plugin outputs events to the file as well."
```

Comment:
```suggestion
    desc "If Fluentd logger outputs logs to a file (with -o option), this plugin outputs events to the file as well."
```

It might be redundant, it helps to understand what it means.

Reply:
Thanks! Applied!